composer.json

```diff
@@ -26,7 +26,8 @@
         "guzzlehttp/psr7": "^1.4",
         "mockery/mockery": "^1.0",
         "php-http/message": "^1.6",
-        "php-http/mock-client": "^1.1"
+        "php-http/mock-client": "^1.2",
+        "phpunit/phpunit": "^7.0"
```

Comment:
Please dont all PHPUnit as a dev-dependency, we prefer using `.phar`.

Reply:
Removed